docs/howto/retransform_datasets.md

```diff
@@ -0,0 +1,29 @@
+# How to re-transform a Dataset
```

Comment:
In fact you're not retransforming DataSet - you're retransforming Run (into DataSet(s))

Reply:
If it adds clarity to what's happening here then that has to be a good thing.